tutorial/Lab2/README.md

```diff
@@ -28,64 +32,70 @@ In this part of the lab we will update the previously deployed application [Gues
    deployment.extensions "redis-slave" deleted
    ```
 
-3. Update the guestbook service from `LoadBalancer` to `NodePort` type:
+1. Update the guestbook service from `LoadBalancer` to `NodePort` type:
 
    ```console
-   $ sed -i.bak 's/LoadBalancer/NodePort/g' guestbook-service.yaml
+   sed -i.bak 's/LoadBalancer/NodePort/g' guestbook-service.yaml
    ```
 
-   Note: Like in Step 1, you may want to archive before making the changes.
-   
-4. Delete the guestbook service:
-    
-    ```console
-    $ kubectl delete svc guestbook --namespace default
-    ```
-    
-5. Re-create the service with `NodePort` type:
+   > Note: you can reset the files later with a `git checkout -- <filemane>` command, if desired
```

Comment:
I think the example is correct. Without the `--` the checkout command argument is presumed to be a branch rather than a filespec. See: https://git-scm.com/docs/git-checkout

Reply:
Same comment above. Just change:
filemane --> filename

Reply:
d'oh ... e2f9fd4